src/Microsoft.DotNet.MSBuildSdkResolver/MSBuildSdkResolver.cs

```diff
@@ -53,14 +53,26 @@ public override SdkResult Resolve(SdkReference sdkReference, SdkResolverContext
                 netcoreSdkVersion = new DirectoryInfo(netcoreSdkDir).Name;;
             }
 
+            if (!string.IsNullOrEmpty(sdkReference.MinimumVersion) && 
+                SemanticVersion.Parse(netcoreSdkVersion) < SemanticVersion.Parse(sdkReference.MinimumVersion))
```

Comment:
True. Wonder what we should do if any of the versions are not formatted right though. In the resolver.